1-Vanilla/vanillaTodoListCopy.js

```diff
@@ -0,0 +1,127 @@
+
+(function () {
+
+const createCheck = (todoId) => {
+    let checkDiv = document.createElement('input');
+    checkDiv.type = 'checkbox'
+    const checkDivClassName = "checked"
+    checkDiv.classList.add(checkDivClassName)
+    checkDiv.onclick = function(){handleCheckedClick(todoId)}
```

Comment:
The downside of inline event such as `element.onclick` and not adding an event listener with `addEventListener('click'...)` is that you may only register one at a time and it can be overriden. 
since you're providing `createCheck` as a function it might be problematic for consumers later on